networking_oneview/ml2/drivers/oneview/common.py

```diff
@@ -404,14 +404,13 @@ def is_lig_id_uplink_name_mapped(lig_bd_entry, mappings):
     return False
 
 
-def get_boot_priority(server_profile, bootable):
-    if bootable:
-        connections = server_profile.get('connections')
-        if _is_boot_priority_available(connections, 'Primary'):
-            return 'Primary'
-        elif _is_boot_priority_available(connections, 'Secondary'):
-            return 'Secondary'
-    return 'NotBootable'
+def get_boot_priority(server_profile):
```

Comment:
I was thinking more of:
 if openstack user asked to create a non-bootable port: `NotBootable`
 if openstack user asked to create a bootable port: `Primary`, `Secondary` or `None` (depending on the availability)

And then we would fail the port creation only if `None` was returned... wdyt?

Reply:
@ricardoas remember to talk with @reinaldo-ufcg about this.

Reply:
We though about it and the conclusion is that the validation is wrong in stopping users from creating non-bootable ports. We should change [there](https://github.com/HewlettPackard/networking-oneview/blob/master/networking_oneview/ml2/drivers/oneview/common.py#L526) and change this implementation too...

Reply:
done